pkgs/os-specific/linux/decklink/default.nix

```diff
@@ -17,9 +18,24 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs =  kernel.moduleBuildDependencies;
 
+  patches = lib.optionals (lib.versionAtLeast kernel.version "6.8") [
+    (fetchpatch {
+      name = "decklink-addMutex.patch";
+      url = "https://aur.archlinux.org/cgit/aur.git/plain/01-addMutex.patch?h=decklink&id=132ce45a76e230cbfec4a3daac237ffe9b8a377a";
```

Comment:
technically could go stale due to force push but i don't think this is an issue.

Reply:
the id in that URL is the commit ID, so that can only happen if the commit is then garbage collected. honestly, very unlikely to happen.